data/human/hails.txt

```diff
@@ -1568,6 +1570,109 @@ phrase "friendly civilian"
 		"Apparently people think we'll work for exposure."
 		"Not to mention this ship is not designed for a job like it."
 
+phrase "friendly civilian"
+	word
+		"Don't go for"
+		"Don't participate in"
+		"Stay away from"
+		"Watch out for"
+	word
+		" "
+	word
+		"badly organized races"
+		"inheritance scams"
+		"shady transport jobs"
+		"hitman transports"
+	word
+		"."
+		"!"
+	word
+		" "
+	word
+		"I lost 20000 credits on one of those."
+		"I nearly got killed in one."
+		"The Navy almost arrested me."
+		"Those can ruin your life!"
+		"They are usually too good to be true."
+		"They take your money and you won't get anything in return."
+
+phrase "friendly civilian"
+	word
+		"I have a delivery for"
+		"I'm looking for"
+		"My passengers need to go to"
+	word
+		" "
+	word
+		"Dune"
+		"Earth"
+		"Foundry"
+		"Glaze"
+		"Moonshake"
+		"Norn"
+		"Skillet"
+	word
+		"; "
+	word
+		"but my starmap doesn't list it."
```

Comment:
This is a dependent clause, not an independent clause.

Reply:
I will update the structure to use a comma instead of a semicolon.